Extensions/TextInput/JsExtension.js

```diff
@@ -572,6 +609,22 @@ module.exports = {
       .getCodeExtraInformation()
       .setFunctionName('isFocused');
 
+    object
+      .addScopedCondition(
+        'IsInputSubmitted',
+        _('Input is Submitted (Enter pressed'),
```

Comment:
```suggestion
        _('Input is submitted'),
```
I think it's better not mentioning any platform specific detail in the name

Reply:
I think it's better without the capital S